src/phpDocumentor/Pipeline/Stage/Configure.php

```diff
@@ -50,7 +51,7 @@ public function __construct(
     public function __invoke(array $options) : array
     {
         $this->configFactory->addMiddleware(
-            new CommandlineOptionsMiddleware($options, $this->configFactory)
+            new CommandlineOptionsMiddleware($options, $this->configFactory, getcwd())
```

Comment:
Consider using the Kernel's `getWorkingDir` method; this is essentially a wrapper around getcwd but is useful for service injection using `@=service('kernel').getWorkingDir()`